lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -1101,318 +1100,3 @@ def format_calibration_method(calibration_method):
-            if np.any(np.isnan(result)):
-                msg = ("NaNs are present within the result for the {} "
-                       "percentile. Unable to calculate the percent point "
-                       "function.")
-                raise ValueError(msg)
-
-        result = result.T
-
-        t_coord = calibrated_forecast_predictor.coord("time")
-        y_coord = calibrated_forecast_predictor.coord(axis="y")
-        x_coord = calibrated_forecast_predictor.coord(axis="x")
-
-        result = result.reshape(
-            len(percentiles), len(t_coord.points), len(y_coord.points),
-            len(x_coord.points))
-        percentile_cube = self._create_cube_with_percentiles(
-            percentiles, calibrated_forecast_predictor, result)
-
-        percentile_cube.cell_methods = {}
-        return percentile_cube
-
-    def _create_percentiles(
-            self, no_of_percentiles, sampling="quantile"):
-        """
-        Function to create percentiles.
-
-        Parameters
-        ----------
-        no_of_percentiles : Int
-            Number of percentiles.
-        sampling : String
-            Type of sampling of the distribution to produce a set of
-            percentiles e.g. quantile or random.
-            Accepted options for sampling are:
-            Quantile: A regular set of equally-spaced percentiles aimed
-                      at dividing a Cumulative Distribution Function into
-                      blocks of equal probability.
-            Random: A random set of ordered percentiles.
-
-        For further details, Flowerdew, J., 2014.
-        Calibrating ensemble reliability whilst preserving spatial structure.
-        Tellus, Series A: Dynamic Meteorology and Oceanography, 66(1), pp.1-20.
-        Schefzik, R., Thorarinsdottir, T.L. & Gneiting, T., 2013.
-        Uncertainty Quantification in Complex Simulation Models Using Ensemble
-        Copula Coupling.
-        Statistical Science, 28(4), pp.616-640.
-
-        Returns
-        -------
-        percentiles : List
-            Percentiles calculated using the sampling technique specified.
-
-        """
-        if sampling in ["quantile"]:
-            percentiles = np.linspace(
-                1/float(1+no_of_percentiles),
-                no_of_percentiles/float(1+no_of_percentiles),
-                no_of_percentiles).tolist()
-        elif sampling in ["random"]:
-            percentiles = []
-            for _ in range(no_of_percentiles):
-                percentiles.append(
-                    random.uniform(
-                        1/float(1+no_of_percentiles),
-                        no_of_percentiles/float(1+no_of_percentiles)))
-            percentiles = sorted(percentiles)
-        else:
-            msg = "The {} sampling option is not yet implemented.".format(
-                sampling)
-            raise ValueError(msg)
-        return percentiles
-
-    def process(self, calibrated_forecast_predictor_and_variance,
```

Comment:
Moved to lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py unchanged